libs/network/tests/muddle/mrpc_stress_tests.cpp

```diff
@@ -142,7 +142,7 @@ class MuddleRpcStressTests : public ::testing::Test
     {
       buffer[i] = fill;
     }
-    return std::move(buffer);
+    return buffer;
```

Comment:
I have seen this pattern a few times with `ConstByteArray` I suggest maybe adding braces to make it clear that a new object is being created. i.e. `return {std::move(buffer)}`.